### ISSUE TITLE
Biosynthetic Gene cluster detection

### ISSUE THREAD

**Issue by iqbal-lab** [iqbal-lab-org/AllTheBacteria#17]
Laura Carroll has volunteered

**Comment by iqbal-lab** [comment]
In progress

**Comment by iqbal-lab** [comment]
Worth talking to Laura about the fact that release 0.2 will change the contigs in 12,000 assemblies (removing human contamination) and how she wants to handle this

**Comment by lmc297** [comment]
It's no problem! If I understand correctly, the 12k contigs will just be omitted from release 0.2? If this is the case, we can just run on everything like we've bee doing and omit results for the 12k contigs afterward.

**Comment by iqbal-lab** [comment]
We'll re-release the full set of assemblies, and withdraw 0.1.
For 2million - 12k assemblies, there will be no change
For 12k assemblies (each consisting of many contigs), some contigs will disappear from each (which may alter contig naming/numbering)
This also will change the high quality set (some formerly low qual actually become high qual, as the N50 improves after discarding contaminating contigs)
Hope this makes sense?
Basically, you will need to reprocess 12k genomes i think, and be very careful about switching in the new versions.

**Comment by lmc297** [comment]
Ah got it, the contig relabeling changes things; no problem though, we'll redo the 12k (and of course be mindful of working with different versions)

**Comment by iqbal-lab** [comment]
How did this go/is this going @lmc297 ?

**Comment by lmc297** [comment]
Great! We finished running on the original ATB submission + now the genomes affected by v0.2; currently running some sanity checks, and we're about to start running on the new 500k+ genomes

**Comment by iqbal-lab** [comment]
Fantastic!!

**Comment by lmc297** [comment]
@iqbal-lab We've finished detecting BGCs in all ATB genomes, just some final sanity checks, and we'll be ready to upload! Per the docs (https://allthebacteria.readthedocs.io/en/latest/contributing.html), I'm seeing that we will need "access to an OSF project component, where [we] can upload results files"; would it be possible to get access? (my apologies if this is something I missed on my end!)

**Comment by iqbal-lab** [comment]
Fantastic news! Am cc-ing @martinghunt who manages OSF access

**Comment by martinghunt** [comment]
Thanks for contributing!

Here you go:
https://osf.io/cufb2/

You’ll need an OSF account, and then use it to request access and I can add you to that component

**Comment by martinghunt** [comment]
@lmc297 I just added you to the OSF component with "read+write" permissions. There's also the option of "administrator" - I can change it to that if you run in to any issues

**Comment by martinghunt** [comment]
I think this is all done (PR #57)? If so we can close this issue...
